api/client/swarm/opts.go

```diff
@@ -112,6 +112,15 @@ func (o *AutoAcceptOption) Set(value string) error {
 			return fmt.Errorf("value NONE is incompatible with %s", value)
 		}
 		o.values[value] = true
+	case "ALL":
+		if accept, ok := o.values[WORKER]; ok && accept {
+			return fmt.Errorf("value NONE is incompatible with %s", WORKER)
+		}
+		if accept, ok := o.values[MANAGER]; ok && accept {
+			return fmt.Errorf("value NONE is incompatible with %s", MANAGER)
```

Comment:
Thanks @aluzzardi, missed these but actually the logic needs to be tightened and can be made simpler now I think about it (after adding `ALL`). Just running tests on my changes and then will update the PR.